src/concepts/glean/glean.md

```diff
@@ -1,46 +1,69 @@
-# Glean - product analytics & telemetry
+# Glean
 
-For Mozilla, getting reliable data from our products is critical to inform our decision making. Glean is our new product analytics & telemetry solution that provides that data for our products.
-It aims to be easy to integrate, reliable and transparent by providing an SDK and integrated tools.
+For Mozilla, getting reliable data from our products is critical to inform our decision making. Glean is our new product analytics & telemetry solution that will provide consistent data behavior and development experience across all of our products.
 
-It currently supports Android and iOS products, while Desktop support is planned.
-Note that this is different from Telemetry for Firefox Desktop ([library](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/index.html), [datasets](../choosing_a_dataset.md)), although it provides similar capabilities.
+It currently supports Android, iOS products, and Python based tools, while support for Desktop Firefox is in development.
```

Comment:
```suggestion
The list of supported platforms and implementations is [available in the Glean SDK Book](https://mozilla.github.io/glean/book/dev/core/internal/implementations.html).
```

Instead of keeping this list, we could point to our other docs, which would hopefully be always consistent :)